utils.py

```diff
@@ -203,10 +203,17 @@ def viewer(sample):
 def prepare(samples):
     print("Preparing data")
 
-    X = []
     y = []
-
+    
     for sample in samples:
+        image_files = load_imgs(sample)
```

Comment:
Hey @mgagvani ! It was pointed out to me in https://github.com/kevinhughes27/TensorKart/issues/66 that this function is missing. Do you happen to have it or remember exactly what it did? I am guessing it loads all the images for the sample which wouldn't be too hard to re-implement but if you have it that would be even easier :)

Reply:
Yeah, I'm 99% sure I have that, when I get back to my computer I'll get back to you.